pkg/version/latest/latest.go

```diff
@@ -47,6 +47,7 @@ func CheckLatest(ctx context.Context) {
 		logger.Log(logger.Gray("  " + getUpgradeCommand()))
 		logger.Log("")
 	}
+	return latestWithoutPrefix == version.Get()
```

Comment:
Already doing the check above^
```suggestion
	return true
```